pkg/features/list.go

```diff
@@ -126,4 +126,7 @@ var (
 
 	// Splits Image CVEs into Workload CVE and Platform CVE sections in the UI
 	PlatformCVESplit = registerFeature("Splits Image CVEs into Workload CVE and Platform CVE sections in the UI", "ROX_PLATFORM_CVE_SPLIT")
+
+	// Flattens CVE Data Model for improved accuracy and performance
+	FlattenCVEData = registerFeature("Uses a flattened CVE Data Model improved accuracy and performance", "ROX_FLATTEN_CVE_DATA")
```

Comment:
Do you foresee any need for the UI to know the status of this flag?

If so, we need to add it to their type enum for TypeScript, in this file as in this example,
https://github.com/stackrox/stackrox/pull/13520/files#diff-5476f398e449ca28ed480251b55c52792343ce347ab4afc0dbd76c37a15f455b

Reply:
I do not at this time.  Leaving that out was intentional, believe it or not.